### PR TITLE
Allow to specify a different database charset via environment variable

### PR DESCRIPTION
**Pull request by signaleleven**
I had to import a database from a very old wordpress instance, created before the standard charset for the database was utf8.
This change came in handy to configure wordpress accordingly.
(when converting the database in not an easy option)

This comes with a sensible default, and it's harmless if not used.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,6 +104,7 @@ To access the Wordpress Admin at `/wp-admin`, the default values are as follows:
...
+| `DB_CHARSET`        | `utf8`                            | Select a charset for the wordpress database (legacy versions might not be utf8)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
```
Comment (karellm) [inline README.md]: Can you make sure the pipes | are aligned, aka that there are no extra space between lines?

**Comment by dsifford** [comment]
No objections. Seems reasonable.

**Comment by signaleleven** [comment]
Sorry, I amended the commit and now this PR is a bit ugly :) But I changed what was requested.

**Comment by karellm** [comment]
@signaleleven  You missed one space before the second pipe :trollface:

**Comment by signaleleven** [comment]
Third time's the charm? :D

**Comment by karellm** [comment]
@signaleleven Thanks for the contribution

@dsifford LGTM, can I ask you to merge and deploy a new version?

**Comment by dsifford** [comment]
@karellm Yeah, I can deploy.

I have one little thing I'm gonna add and PR this morning though that's separate from this PR. I'd prefer to push both updates at the same time if possible.

**Pull request merged**